### DISCUSSION TITLE
batch rendering options for moonray?

### DISCUSSION THREAD

**Discussion by mpjohn** [dreamworksanimation/openmoonray#150] [General]
I would like to send distributed renders to multiple workstations with Moonray installed.
What's the simplest way of going about this?
Through Houdini / Deadline? 
What if I only want to send command line renders?
Any help on this would be much appreciated.

**Comment by jlanz** [comment]
Batch rendering with MoonRay is done with the [moonray](https://docs.openmoonray.org/user-reference/tools/moonray/) cmd line tool.  This requires that the scenes be in MoonRay's native [RDL2](https://docs.openmoonray.org/getting-started/about/rdl-scene-format/) scene format.

If you are wanting to batch render USD scenes directly with MoonRay from Houdini without translating the scenes to RDL2 format you will likely need to use something like [husk](https://www.sidefx.com/docs/houdini/ref/utils/husk.html) to invoke the MoonRay's hydra render delegate, although I am probably not the right person to offer much guidance on this.  Maybe someone more knowledgable about this will chime in.

I can't really recommend any particular render farm management system, but probably any would work.
